Octokit/Models/Request/SearchUsersRequest.cs

```diff
@@ -117,7 +117,7 @@ public override IReadOnlyList<string> MergedQualifiers()
 
             if (Language != null)
             {
-                parameters.Add(string.Format(CultureInfo.InvariantCulture, "language:\"{0}\"", Language));
+                parameters.Add(string.Format(CultureInfo.InvariantCulture, "language:\"{0}\"", Language.ToParameter()));
```

Comment:
Actually found that this part didn't use to Parameter so named languages such as CPlusPlus was parsed as "CPlusPlus" instead of "cpp"